internal/pkg/model/naming.go

```diff
@@ -14,10 +14,15 @@ const (
 	MetaFile          = "meta.json"
 	ConfigFile        = "config.json"
 	DescriptionFile   = "description.md"
+	PhaseFile         = "phase.json"
```

Comment:
Bude jasný, že ty konstanty patří k Orchestrátoru? Nebo to není potřeba?

Reply:
rozmyslal som nad tym, ... dam tam komentar? nechcel som zase vsetko prefixovat s `Orchestrator`, .. a tie nazvy su unikatne, ... ono by to asi mohlo byt aj priamo v `orchestrator` balicku ... este sa na to pozriem, ked to dokoncim, ze ake vsetky zavislosti tam budu. Dik.